src/ts/mapboxMap.ts

```diff
@@ -84,7 +84,7 @@ module powerbi.extensibility.visual {
                     });
                     mapboxUtils.zoomToData(map, bounds, this.autoZoomControl.isPinned());
                 }
-                if (!geocoder) {
+                if (!settings.api.geocoder) {
                     // var geoElement = document.querySelector('.')
                     document.querySelector('.geoDiv').classList.add('hidden')
```

Comment:
One last thought - should we make the geocoder div an HTMLElement a class variable in `MapboxMap.ts`?  That would follow the same convention we used for `draw` and `autozoom toggles`.  @samgehret